FLEDGE_browser_bidding_and_auction_API.md

```diff
@@ -124,6 +124,54 @@ The browser verifies it witnessed a Fetch request to the `seller`’s origin wit
 
 ## Device Orchestrated Multi-Seller Auctions
 
+Bidding and Auction Services auctions can also be used as component auctions. It
+is possible for multiple Bidding and Auction Services auctions to be used as
+component auctions for an on-device top-level auction.
+
+For efficiency, a top-level seller can construct encrypted requests for multiple
+component sellers in a single call to `navigator.getInterestGroupAdAuctionData()`:
+
+```javascript
+const result = await navigator.getInterestGroupAdAuctionData({
```

Comment:
Sample code nit: either `result` or `results`, here and below.

Reply:
Good catch! Thanks.